.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+<!--
+If your issue is a usage question, submit it here instead:
+- The imbalanced learn gitter: https://gitter.im/mne-tools/mne-python
+-->
+
+<!-- Instructions For Filing a Bug: https://martinos.org/mne/stable/faq.html#i-think-i-found-a-bug-what-do-i-do -->
+
+#### Description
+<!-- Example: Joblib Error thrown when calling fit on LatentDirichletAllocation with evaluate_every > 0-->
+
+#### Steps/Code to Reproduce
+<!--
+Example:
```

Comment:
idem, could I get an MWE related that reflects the description.

Maybe a link or short description of MWE should be provided there. WDYT?